metaflow/plugins/cards/card_modules/components.py

```diff
@@ -103,15 +118,17 @@ def __init__(self, data=[[]], headers=[]):
             self._data = data
 
     @classmethod
-    def from_dataframe(cls, dataframe=None, truncate=True):
+    def from_dataframe(
+        cls, dataframe: Optional["DataFrame"] = None, truncate: bool = True
+    ):
         """
         Create a `Table` based on a Pandas dataframe.
 
         Parameters
         ----------
-        dataframe : pandas.DataFrame
+        dataframe : pandas.DataFrame, optional
```

Comment:
`dataframe=None` is a bit incorrect in the function signature - `dataframe` is not `optional` here